packages/jazz-tools/src/inspector/viewer/table-viewer.tsx

```diff
@@ -82,29 +87,47 @@ function CoValuesTableView({
         <TableBody>
           {resolvedRows.slice(0, visibleRowsCount).map((item, index) => (
             <TableRow key={index}>
-              {keys.map((key) => (
-                <TableCell key={key}>
-                  <ValueRenderer
-                    json={(item.snapshot as JsonObject)[key]}
-                    onCoIDClick={(coId) => {
-                      async function handleClick() {
-                        onNavigate([
-                          {
-                            coId: item.value!.id,
-                            name: index.toString(),
-                          },
-                          {
-                            coId: coId,
-                            name: key,
-                          },
-                        ]);
-                      }
-
-                      handleClick();
-                    }}
-                  />
+              <TableCell>
+                <Text mono>
+                  {item.snapshot === "unavailable" && (
+                    <Icon
+                      name="caution"
+                      color="red"
+                      style={{ display: "inline-block", marginRight: "0.5rem" }}
+                    />
+                  )}
+                  {visibleRows[index]}
```

Comment:
Right now we show nothing if the colist contains non-collaborative values, right? I wonder if that could be confusing